opencti-worker/src/worker.py

```diff
@@ -439,6 +440,16 @@ def __post_init__(self) -> None:
         self.log_level = get_config_variable(
             "WORKER_LOG_LEVEL", ["worker", "log_level"], config
         )
+        # Custom authentication
+        self.custom_headers = get_config_variable(
+            "CUSTOM_HEADERS", ["opencti", "custom_headers"], config
```

Comment:
Could you give a use case example of custom header ? I don't see custom "CUSTOM_HEADERS" usage in OpenCTI, so I have difficulties to see how to tests this part.

Reply:
Hi @aHenryJard,

Thanks for your reply. A potential use case is as follows:

- You're hosting the OpenCTI Platform and its Workers on a 'serverless' environment at a cloud vendor, e.g. Google Cloud's Cloud Run or Azure's Container Apps.
- Serverless environments may contain multiple authentication methods/settings, for instance an Identity-Aware Proxy.
- For instance, GCP Cloud Run expects an extra header beside the traditional `Authorization: Bearer ID_TOKEN`, namely the `X-Serverless-Authorization: Bearer ID_TOKEN`. This described [here](https://cloud.google.com/run/docs/authenticating/service-to-service#acquire-token).
- Therefore, you may want to influence the way how the Worker sends HTTP request to the Platform.